### PR TITLE
Create stale-issues-pr.yml

### PR DESCRIPTION
**Pull request by sachinprasadhs**
This action workflow is to close the inactive issues and PRs which is awaiting response from the contributor.

### CODE REVIEW

File: .github/workflows/stale-issues-pr.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Awaiting response issues
+        uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 7
```
Comment (fchollet) [inline .github/workflows/stale-issues-pr.yml:16]: This seems too low, let's use 14

File: .github/workflows/stale-issues-pr.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          stale-issue-label: "stale"
+          close-issue-reason: completed
+          only-labels: "stat:awaiting response from contributor"
+          stale-issue-message: > 
+            This issue is stale because it has been open for 7 days with no activity.
+            It will be closed if no further activity occurs. Thank you.
+          close-issue-message: >
+            This issue was closed because it has been inactive for 14 days.
+            Please reopen if you'd like to work on this further.
+          days-before-pr-stale: 14
```
Comment (fchollet) [inline .github/workflows/stale-issues-pr.yml:27]: Let's use 30?

File: .github/workflows/stale-issues-pr.yml (hunk 1 above)
Comment (sachinprasadhs) [reply .github/workflows/stale-issues-pr.yml:16]: To apply stale label  it's 7 days, to close the issue 7 more days, total 14 days.

File: .github/workflows/stale-issues-pr.yml (hunk 2 above)
Comment (sachinprasadhs) [reply .github/workflows/stale-issues-pr.yml:27]: To apply stale label  it's 14 days, to close the issue 14 more days, total 28 days.

File: .github/workflows/stale-issues-pr.yml (hunk 1 above)
Comment (fchollet) [reply .github/workflows/stale-issues-pr.yml:16]: Still seems too low -- let's go for 14+14

**Review by fchollet — APPROVED** [review]

**Pull request merged**